addons/web_editor/static/src/js/wysiwyg/widgets/link.js

```diff
@@ -106,13 +106,19 @@ const Link = Widget.extend({
             this.data.isNewWindow = this.data.isNewWindow || this.linkEl.target === '_blank';
         }
 
+        const classesToKeep = [
+            'btn-block', 'text-wrap', 'text-nowrap', 'text-left', 
+            'text-center', 'text-right', 'text-justify', 'text-truncate',
```

Comment:
Are those the only bootstrap text-* classes that exists?

If I understand correctly, the original regex which aimed at removing color related classes, also removed utility and text alignment related classes, right? 

I am wondering why the regex exists in the first place, why remove classes if we do not apply custom colors to a link?
Wouldn't it be worth to only filter out the color classes when we are applying a new color?

Reply:
@bvr-odoo Since there is a task (3114415) planned to refactor the way the link tools interact with the DOM, I'm ok with this approach for now, as I doubt that we will add more classes on links in stable (🤞🏻).

However I would still export the classesToKeep array to allow custom modules to add their own class, what do you think?

Reply:
> @bvr-odoo Since there is a task (3114415) planned to refactor the way the link tools interact with the DOM, I'm ok with this approach for now, as I doubt that we will add more classes on links in stable (🤞🏻).
> 
> However I would still export the classesToKeep array to allow custom modules to add their own class, what do you think?

Hello @detrouxdev , I tested yesterday on a v15 of the runbot and it seems that the bug no longer exists. I'll check it better on Monday, but I have a feeling we can cancel this PR.

Reply:
It seems like https://github.com/odoo/odoo/commit/9cd8c6dffcbc1a18cfc85e632d315720361e4df1 fixed the card breaking when clicking on it, but when editing the link with the link tools (by clicking on the text), the card still breaks. So this PR might still be necessary.

Note that we can still update the links in this layout by clicking on the image instead of the link tools. Though not ideal, it's at least a workaround for now.

Reply:
> However I would still export the classesToKeep array to allow custom modules to add their own class, what do you think?

@detrouxdev
I don't think we should encourage to add classes on links which are editable (especially in stable version). If it really needs to be done for a custom module they can always (at worst) add them here while waiting for the refactoring in master.

Reply:
> Are those the only bootstrap text-* classes that exists?

There are others (like "btn-lg") but they are not used by link tools so there is no need to put them here.

Reply:
> I don't think we should encourage to add classes on links which are editable (especially in stable version). If it really needs to be done for a custom module they can always (at worst) add them here while waiting for the refactoring in master.

Though I agree that we shouldn't encourage it, I think it can happen quickly to just add a custom class on a template like a mega menu or a snippet from a theme. Editing the link would break the style. But I'll trust your judgement 😄 it's fine like this.